metadata.json

```diff
@@ -125,6 +125,6 @@
     {"name":"puppetlabs-stdlib","version_requirement":">= 5.1.0 < 7.0.0"},
     {"name":"puppetlabs-inifile","version_requirement":">= 2.4.0 < 5.0.0"},
     {"name":"puppetlabs-apt","version_requirement":">= 7.0.1 < 8.0.0"},
-    {"name":"puppetlabs-facts","version_requirement":">= 0.5.0 < 1.0.0"}
+    {"name":"puppetlabs-facts","version_requirement":">= 0.5.0"}
```

Comment:
Not sure if I should limit this to the next major release of puppetlabs-facts `">= 0.5.0 < 2.0.0"`

Reply:
Should be locked to `< 2.0.0"` as we want to control the next version major bump